zeppelin-web/src/app/notebook/notebook-actionBar.html

```diff
@@ -15,7 +15,7 @@
   <h3>
     <div style="float: left; width: auto; max-width: 40%"
       ng-controller="ElasticInputCtrl as input">
-      <input type="text" pu-elastic-input class="form-control2" placeholder="{{noteName(note)}}" style="min-width: 0px; max-width: 95%;"
+      <input type="text" pu-elastic-input class="form-control2" placeholder="Note {{note.id}}" style="min-width: 0px; max-width: 95%;"
```

Comment:
I understood this fix. 
But displaying a note id in the placeholder seems to be weird also, isn't it? 
The placeholder is just a simple value or a short description of the expected input.